conan/tools/cmake/toolchain/blocks.py

```diff
@@ -919,6 +920,65 @@ def _is_apple_cross_building(self):
         return os_host in ('iOS', 'watchOS', 'tvOS', 'visionOS') or (
                 os_host == 'Macos' and (arch_host != arch_build or os_build != os_host))
 
+    def _get_darwin_version(self, system_name, os_version):
+        darwin_versions = {
+            "macos_old": {
+                "10.13": "17",
+                "10.14": "18",
+                "10.15": "19"
+            },
+            "macos_new": {
+                "11": "20",
+                "12": "21",
+                "13": "22",
+                "14": "23",
+            },
+            "iOS": {
```

Comment:
This could be changed to `"iOS": {str(i): str(6 + i) for i in range(11, 18)` but i think it makes it less clear which versions we are supporting.